skel.ebuild

```diff
@@ -89,11 +91,15 @@ IUSE="gnome X"
 # The below is valid if the same run-time depends are required to compile.
 RDEPEND="${DEPEND}"
 
+# Build-time dependencies that are executed during the emerge process, and 
+# only need to be present in the host system, rather than target. Example:
+#BDEPEND="virtual/pkgconfig"
```

Comment:
We avoided "host system" in PMS, because it could be confused with CHOST. So better say something like "native build system".

Reply:
Will update, cheers :)

Reply:
It's surprisingly hard to explain BDEPEND shortly. I may want to add few lines to clarify it a bit tomorrow, or should link to PMS's BDEPEND section (8.2) be provided here in your opinion? I'd like to get it more beginner-friendly.